### ISSUE TITLE
Font status

### ISSUE THREAD

**Issue by videosmith** [Cuperino/QPrompt#195]
**Is your feature request related to a problem or a limitation? Please describe...**
When selectting text, no immediate indication of the font is displayed

**Describe the solution you'd prefer**
Display font of current selected text at the top of the choose font window.

**Comment by Cuperino** [comment]
In v1.2, if the font selection window is opened after text is selected, the font used by the current selected text is automatically selected; if no text is selected at the time the window is open, it shows the font used by the character to the right of the cursor.

What you're suggesting is for the text selection window to be updated with new text selections done after it was opened. Did I understand this correctly?

**Comment by videosmith** [comment]
Yes, if the window listing embedded fonts could include the current selected at the top like word processors do, instead of selecting System Fonts to reveal the current font in the requestor window, would be nice.

    On Sunday, July 16, 2023 at 05:44:37 PM EDT, Javier O. Cordero Pérez ***@***.***> wrote:  
 
 


In v1.2, if the font selection window is opened after text is selected, the font used by the current selected text is automatically selected; if no text is selected at the time the window is open, it shows the font used by the character to the right of the cursor.

What you're suggesting is for the text selection window to be updated with new text selections done after it was opened. Did I understand this correctly?

**Comment by Cuperino** [comment]
Would this work for you? I could also have in the toolbar itself, but space there is at a premium.

<img width="1130" height="477" alt="Image" src="https://github.com/user-attachments/assets/9dea4e80-7389-4b37-9140-7403bfb49560" />

See where it says "Active font: name of font"

**Comment by videosmith** [comment]
i vote yes

**Comment by Cuperino** [comment]
Done. Made it bold to improve contrast and merged into the 2.1 branch.
